Marlin/SanityCheck.h

```diff
@@ -983,3 +983,59 @@
 #if COUNT_LCD_24 > 1
   #error "Please select no more than one LCD controller option."
 #endif
+
+
+#if ENABLED( SPINDLE_ENABLE)
```

Comment:
Spacing in `SanityCheck.h` still needs a conscientious cleanup.
It looks like it was typed out by a dolphin wearing mittens.